### DISCUSSION TITLE
Exporting current taxonomy and keywords

### DISCUSSION THREAD

**Discussion by dlglin** [openwebwork/webwork-open-problem-library#835] [General]
The WeBWorK wiki currently links to http://hobbes.la.asu.edu/Holt/chaps-and-secs.html and http://hobbes.la.asu.edu/Holt/keywords.html, which I believe that @jwj61 was maintaining.  Are these still being maintained?  Can we get github to automatically generate these documents?

**Comment by jwj61** [comment]
No, they are not being maintained.
